pyanaconda/modules/storage/devicetree/fsset.py

```diff
@@ -530,7 +530,7 @@ def turn_on_swap(self, root_path=""):
                 try:
                     device.setup()
                     device.format.setup()
-                except blockdev.SwapActivateError as e:
+                except (SwapSpaceError, blockdev.SwapActivateError) as e:
```

Comment:
The `SwapSpaceError` is not new so this is backwards compatible and there's no need to wait for blivet 3.8.0 release.